release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -71,9 +71,34 @@ This release adds improvements related to the following components and concepts:
 [id="ocp-release-notes-ibm-z_{context}"]
 === {ibm-z-title} and {ibm-linuxone-title}
 
+With this release, {ibm-z-name} and {ibm-linuxone-name} are now compatible with {product-title} {product-version}. You can perform the installation with z/VM, LPAR, or {op-system-base-full} Kernel-based Virtual Machine (KVM). For installation instructions, see
+xref:../installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc#preparing-to-install-on-ibm-z[Preparing to install on {ibm-z-title} and {ibm-linuxone-title}].
```

Comment:
```suggestion
xref:../installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc#preparing-to-install-on-ibm-z[Installation methods].
```
the actual title of the section

Reply:
Agree we changed the title with the restructuring.